contracts/round/RoundImplementation.sol

```diff
@@ -368,9 +383,54 @@ contract RoundImplementation is AccessControlEnumerable, Initializable {
     require(
       applicationsStartTime <= block.timestamp  &&
       block.timestamp <= applicationsEndTime,
-      "Round: Applications period over"
+      "Round: Applications period not started or over"
     );
-    emit NewProjectApplication(projectID, newApplicationMetaPtr);
+    applications.push(Application(projectID, nextApplicationIndex, newApplicationMetaPtr));
+    applicationsIndexesByProjectID[projectID].push(nextApplicationIndex);
+    emit NewProjectApplication(projectID, nextApplicationIndex, newApplicationMetaPtr);
+    nextApplicationIndex++;
+  }
+
+  /// @notice Get all applications of a projectID
+  /// @param projectID unique hash of the project
+  /// @return applicationIndexes indexes of the applications
+  function getApplicationIndexesByProjectID(bytes32 projectID) external view returns(uint256[] memory) {
+    return applicationsIndexesByProjectID[projectID];
+  }
```

Comment:
I would leave a blank line here between the function and the next comments

Reply:
changed